plugins/USBPrinting/USBPrinterOutputDevice.py

```diff
@@ -313,6 +313,7 @@ def _update(self):
         while self._connection_state == ConnectionState.connected and self._serial is not None:
             try:
                 line = self._serial.readline()
+                print(line)
```

Comment:
I guess that's a debug print that sneaked along with the commit.

Reply:
'doh!